app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class MenusController < ApplicationController
+  def update
+    @menu = Menu.find(params[:id])
+    recipe = Recipe.find(params[:recipe_id])
+
+    if @menu && recipe
```

Comment:
`find` will actually fail with an exception if @menu and recipe isn't found, so doing `if @menu && recipe` is unnecessary. Good thinking though :)